### DISCUSSION TITLE
How about renaming the class name row/col to q-row/q-col ?

### DISCUSSION THREAD

**Discussion by dioscoreae** [quasarframework/quasar#9766] [Ideas / Proposals] [answered]
Hi,  

The CSS class row and col are very common and can easily conflict with the other library , such as bootstrap.    How about adding a prefix to the class name , such as q-row/q-col  ?

**Comment by rstoenescu** [comment, marked as answer]
Hi,

We are aware about this and have discussed it internally and externally months ago. It's a good idea and we would have done it already in Quasar v2, but it would have been a too heavy breaking change along with Vue 3 and Webpack 5. It might not mean much to small apps, but big apps would have really suffered in the process of upgrading where devs would have needed to simultaneously apply changes in multiple areas just to have a barely "working" project. We promised a smooth upgrade to Qv2 (or as smooth as humanly possible) and we kept our word.

Bottom line, this is scheduled for Qv3 now.
